src/Spago/Dhall.hs

```diff
@@ -50,14 +50,15 @@ readRawExpr pathText = do
     else (pure Nothing)
 
 
-writeRawExpr :: Text -> (Text, DhallExpr Dhall.Import) -> IO ()
-writeRawExpr pathText (header, expr) = do
+writeRawExpr :: Bool -> Text -> (Text, DhallExpr Dhall.Import) -> IO ()
```

Comment:
Could we have a small datatype like `data DoFormat = DoFormat | NoFormat` instead of a `Bool`? (for reasons why, see [boolean blindness](https://github.com/quchen/articles/blob/master/algebraic-blindness.md))

You can see an example of how to read it from CLI [here](https://github.com/spacchetti/spago/blob/a77b869edcfddd592f45726400b816de8c1f070e/app/Spago.hs#L122)

Reply:
Thanks for the pointers. The article was an important read. :smile: . I have changed the implementation. :+1: